mumblebackend/urls.py

```diff
@@ -20,4 +20,5 @@
     path('admin/', admin.site.urls),
     path('api/users/', include('users.urls')),
     path('api/mumbles/', include('feed.urls')),
+    path('api/article/', include('article.urls')),
```

Comment:
I'd make this follow the plural approach we've been using by renaming to `/api/articles`

Reply:
Yes , let me fix